File: Modul-4/Soal-4/PRAK404-2410817210021-MuhammadNafisPutra.py
while (True) :
     print ("1. penjumlahan")
     print("2. Pengurangan")
     print("3. Perkalian")
     print("4. Pembagian")
     print("5. Exit")
     pilihan = int(input("Masukkan Pilihan :"))

     if (pilihan == 5):
        print("Terimakasih, telah menggunakan kalkulator Muhammad Nafis Putra")
        break
     if (pilihan > 5 or pilihan < 1):
        print("Input anda salah, silahkan coba lagi")
        continue

     a = float (input("Masukkan Nilai Pertama :"))
     b = float (input("Masukkan Nilai Kedua :"))

     if pilihan == 1 :
        print(f"Hasil Penjumlahan antara {a : .2f} dengan {b : .2f} adalah {a + b : .2f}")
     if pilihan == 2:
        print(f"Hasil Pengurangan antara {a : .2f} dengan {b : .2f} adalah {a - b : .2f}")
     if pilihan == 3:
        print(f"Hasil Perkalian antara {a : .2f} dengan {b : .2f} adalah {a * b : .2f}")
     if pilihan == 4:
        print(f"Hasil Pembagian antara {a : .2f} dengan {b : .2f} adalah {a / b : .2f}")

File: Modul-4/Soal-4/PRAK404-2410817210021-MuhammadNafisPutra.c
#include <stdio.h>

int main(){
    int pilihan;
    float a, b;

    while(1){
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan :");
        scanf("%d", &pilihan);

        if (pilihan == 5){
        printf("Terimakasih, telah menggunakan kalkulator Muhammad Nafis Putra\n");break;}
        if(pilihan > 5 || pilihan < 1){
        printf("Input anda salah, silahkan coba lagi\n");continue;}
    
    printf("masukkan Nilai Pertama :");
    scanf("%f", &a);
    printf("Masukkan Nilai Kedua :");
    scanf("%f", &b);

    switch(pilihan){
        case 1:
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", a, b, a + b);break;
        case 2:
        printf("Hasil Pengurangan antara %.2f dengan %2.2f adalah %.2f\n", a, b, a - b);break;
        case 3:
        printf("Hasil Perkalian antara %.2f dengan %2.2f adalah %.2f\n", a, b, a * b);break;
        case 4:
        printf("Hasil Pengurangan antara %.2f dengan %2.2f adalah %.2f\n", a, b, a / b);break;
    }
    }
}